science/CubicSDR/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+github.setup        cjcliffe CubicSDR c27e1e65140f5968ad0df65ce25a6e94580517d2
+
+name                CubicSDR
+version             20190319
+categories          science comms
+platforms           darwin macosx
+license             GPL-2
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
```

Comment:
go ahead and add me as comaintainer to this & any of the SDR oriented ports you're working on

Reply:
done